content/modules/userprofile-sync/pages/userprofile_sync.adoc

```diff
@@ -433,11 +422,11 @@ docker stop sync-gateway && docker rm sync-gateway
 You should see configuration files for the latest major version and the previous major version in this folder -- see: <<tbl-config-files>>.
 Choose an appropriate version.
 
+[discrete]
+===== For non-Windows Systems
 
-For non-Windows Systems::
-+
-[{tabs}]
-=====
+[tabs]
```

Comment:
We've used [{tabs}] 
throughout the SDK and Cloud docs.
https://docs.couchbase.com/home/contribute/tabs.html#set-and-create-tabs

Reply:
It doesn't actually need the attribute; [tabs] will work fine as it's a standard Asciidoc feature.

Still, I can put them back for consistency.